airmozilla/main/templates/main/event.html

```diff
@@ -334,6 +335,24 @@ <h1 class="entry-title">
         </div>
       {% endif %}
 
+      {% if event.is_public() %}
+        <div class="tab">
+          <input type="radio" id="tab-share" name="tab-group-1">
+          <label for="tab-share">Share</label>
+          <div class="content">
+            <div>
```

Comment:
Why use individual `<div>` tags and not `<p>`? The paragraph tag feels more semantic. Is there a particular reason why `<div>` is better?